src/daemon/rpmostreed-deployment-utils.cxx

```diff
@@ -786,6 +786,7 @@ rpmostreed_update_generate_variant (OstreeDeployment *booted_deployment,
               auto state = rpmostreecxx::query_container_image_commit (*repo, current_checksum);
               container_changed
                   = rpmostreecxx::deployment_add_manifest_diff (*dict, state->cached_update_diff);
+              g_variant_dict_insert (dict, "version", "s", state->version.c_str ());
```

Comment:
A bit awkward; this section is concerned with the rpmdb diff handling. I think ideally we'd instead just call `query_container_image_commit()` earlier in this function and inject it around the same place where we currently inject it for the OStree case.

Fine as is though!

Reply:
Will take a look at that.

Reply:
Meh, I wouldn't worry about it. If you have to touch this code again, then it might be worth cleaning up a bit.

Reply:
Well, looks like this version is not the correct one (it displays the one from the currently booted deployment) so I'll have to dig into it.